libraries/llvm/text/string.effekt

```diff
@@ -57,16 +57,16 @@ def indexOf(str: String, start: Int, substr: String): Option[Int] = {
 }
 // """ // fix some parsing bug for vscode syntax highlighting
 
-def show(s: String): String =
+def showQuoted(s: String): String =
  "\x22" ++ map(s){ c =>
    if(c == "\x22") {
     "\\\x22"
    } else if (c == "\\") {
     "\\\\"
    } else c
  } ++ "\x22"
-def show(l: List[String]): String = showList(l){e => show(e)}
-def show(o: Option[String]): String = showOption(o){ e => show(e) }
+def showQuoted(l: List[String]): String = showList(l){e => showQuoted(e)}
+def showQuoted(o: Option[String]): String = showOption(o){ e => showQuoted(e) }
```

Comment:
No idea if somebody depends on this, but without this change, I get errors as we define `show` for lists in `immutable/list` and `show` for options in `immutable/option`.
cc @phischu